src/web/components/ClickToView.tsx

```diff
@@ -117,11 +125,12 @@ export const ClickToView = ({
 	const textSize = roleTextSize(role);
 
 	if (
-		shouldDisplayOverlay(
+		shouldDisplayOverlay({
 			isTracking,
 			isOverlayClicked,
-			isInABTestVariant(abTests),
-		)
+			isInABTestVariant: isInABTestVariant(abTests),
+			isMainMedia,
+		})
```

Comment:
I converted this function to accept an object to prevent errors/confusion with prop order now that it is accepting more and more props